rosdep/python.yaml

```diff
@@ -6390,6 +6390,12 @@ python3-future:
   openembedded: [python3-future@meta-python]
   rhel: ['python%{python3_pkgversion}-future']
   ubuntu: [python3-future]
+python3-gdal:
+  debian: [python3-gdal]
+  fedora: [python3-gdal]
+  opensuse: [python3-GDAL]
```

Comment:
Consider also adding the gentoo package: [sci-libs/gdal](https://packages.gentoo.org/packages/sci-libs/gdal).
They have one package for both the c++ and python library